features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/routing/common.jag

```diff
@@ -0,0 +1,17 @@
+<%
+var log = new Log("Common Router");
+
+var uri = request.getRequestURI();
+var filePath = uri.split(request.getContextPath());
+
+log.info(uri.split(request.getContextPath()));
+
+var path = "../public/pages/index.html";
```

Comment:
Let's stick into one type of quotes either single quotes or double quotes. Single quotes would be better.